<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainAdminWindow</class>
 <widget class="QWidget" name="TrainAdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>982</width>
     <height>628</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/trainFrameNew.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>356</x>
     <y>22</y>
     <width>126</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/tariffsButton2New.png</normaloff>:/tariffsButton2New.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>126</width>
     <height>37</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>498</x>
     <y>22</y>
     <width>126</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/stationsButton2New.png</normaloff>:/stationsButton2New.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>126</width>
     <height>37</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>958</x>
     <y>14</y>
     <width>17</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/closeIcon.png</normaloff>:/closeIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>17</width>
     <height>17</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>22</y>
     <width>126</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/homeButton2_New.png</normaloff>:/homeButton2_New.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>126</width>
     <height>37</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>22</y>
     <width>134</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/routeButton2New.png</normaloff>:/routeButton2New.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>134</width>
     <height>37</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>934</x>
     <y>14</y>
     <width>17</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/swapIcon.png</normaloff>:/swapIcon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>17</width>
     <height>17</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>111</x>
     <y>419</y>
     <width>240</width>
     <height>82</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/appendAdminButton_1.png</normaloff>:/appendAdminButton_1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>240</width>
     <height>82</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>373</x>
     <y>419</y>
     <width>240</width>
     <height>82</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/changeAdminButton.png</normaloff>:/changeAdminButton.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>240</width>
     <height>82</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_11">
   <property name="geometry">
    <rect>
     <x>635</x>
     <y>419</y>
     <width>240</width>
     <height>82</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/removeAdminNewButton.png</normaloff>:/removeAdminNewButton.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>240</width>
     <height>82</height>
    </size>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>115</y>
     <width>890</width>
     <height>280</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView {
    border: 3px solid #6E93FC;
    border-radius: 5px;
    gridline-color: #6E93FC; /* #6D55FF */
    background-color: white;	
}

QTableView::item {
    background-color: white;
    font-family: Karla; /* Установка шрифта */
    font-size: 15pt; /* Установка размера шрифта */
    color: black;
}

QTableView::item:selected {
    background-color: #6E93FC; /* #6D55FF */
    color: white;
}

QTableView::item:focus {
    background-color: #6E93FC; /* #6D55FF */
    color: white;
}

QHeaderView::section {
    background-color: #6E93FC; /* #6D55FF */
    color: white;
    font-family: Karla;
    font-size: 14pt;
}

QHeaderView::section:horizontal {
    border: none; /* убираем горизонтальные линии разделителя */
}

QHeaderView::section:vertical {
    border: none; /* убираем вертикальные линии разделителя */
}

QScrollBar:vertical {
    border: none;
    background: white; /* Можно изменить на цвет фона таблицы или окна */
    width: 7px; /* Уменьшенная ширина */
    margin: 0px; /* Если не нужно отступление от краев */
    /*border-radius: 5px; /* Уменьшенное закругление */
}

QScrollBar::handle:vertical {
    background: #6E93FC;
    min-height: 7px; /* Уменьшенная минимальная длина */
    /*border-radius: 5px; /* Уменьшенное закругление */
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: white; /* Это изменит цвет фона неактивных частей ползунка */
}

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
    border: none;
    background: none;
    width: 0px;
    height: 0px;
}

QScrollBar:horizontal {
    border: none;
    background: white; /* Можно изменить на цвет фона таблицы или окна */
    height: 7px; /* Уменьшенная высота */
    margin: 0px; /* Если не нужно отступление от краев */
    /*border-radius: 5px; /* Уменьшенное закругление */
}

QScrollBar::handle:horizontal {
    background: #6E93FC;
    min-width: 10px; /* Уменьшенная минимальная длина */
    /*border-radius: 5px; /* Уменьшенное закругление */
}

QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal {
    background: white; /* Это изменит цвет фона неактивных частей ползунка */
}</string>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_12">
   <property name="geometry">
    <rect>
     <x>214</x>
     <y>22</y>
     <width>126</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 20px; </string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/trainStationButton2.png</normaloff>:/trainStationButton2.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>126</width>
     <height>37</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
